doc/source/whatsnew/v1.2.0.rst

```diff
@@ -314,6 +314,7 @@ Groupby/resample/rolling
 - Bug when subsetting columns on a :class:`~pandas.core.groupby.DataFrameGroupBy` (e.g. ``df.groupby('a')[['b']])``) would reset the attributes ``axis``, ``dropna``, ``group_keys``, ``level``, ``mutated``, ``sort``, and ``squeeze`` to their default values. (:issue:`9959`)
 - Bug in :meth:`DataFrameGroupby.tshift` failing to raise ``ValueError`` when a frequency cannot be inferred for the index of a group (:issue:`35937`)
 - Bug in :meth:`DataFrame.groupby` does not always maintain column index name for ``any``, ``all``, ``bfill``, ``ffill``, ``shift`` (:issue:`29764`)
+- Bug in :meth:`Series.groupby.rolling` compressed number of levels of :class:`MultiIndex` in input to one (:issue:`36018`)
```

Comment:
would be ok with doing this for 1.1.2 if you want to change the whatsnew note

Reply:
Yes of course, I moved the whats new entry